qualtricssurvey/qualtricssurvey.py

```diff
@@ -5,7 +5,7 @@
 import cgi
 import pkg_resources
 
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ugettext as _
```

Comment:
@dcadams This is effectively a revert of a0d8e23..

The change was originally made to fix a Django error that was introduced during the Ginkgo release (presumably a Django upgrade).
Given that, can you provide additional context on why this is needed and what the impact of the change is, specifically in relation to the possibility of regression to the original issue?